connection_pool/connection_pool.go

```diff
@@ -266,8 +268,20 @@ func (connPool *ConnectionPool) Call(functionName string, args interface{}, user
 	return conn.Call(functionName, args)
 }
 
+// Call16 calls registered Tarantool function.
+// It uses request code for Tarantool 1.6, so result is converted to array of arrays.
+// Deprecated since 1.7.4.
```

Comment:
I mentioned this above, but I'll remember just in case: the new IPROTO command (and the deprecation of the old one) was in 1.7.2, not 1.7.4. See https://github.com/tarantool/tarantool/releases/tag/1.7.2.

Reply:
BTW, it also seems worthful to say 'Deprecated since tarantool 1.7.2', not just 1.7.2, because the latter could be misunderstood as the version of the connector.

Reply:
Updated.